src/applications/gi/components/profile/Calculator.jsx

```diff
@@ -61,7 +62,7 @@ export class Calculator extends React.Component {
         </button>
         {expanded ? (
           <div className="form-expanding-group-open">
-            <EligibilityForm />
+            <EligibilityForm eligibilityChange={this.props.eligibilityChange} />
```

Comment:
I think you can actually add `eligibilityChange` directly to `EligibilityForm` without passing it as a prop.

Reply:
`LandingPage` needed to pass in its `handleEligibilityChange` to 
`eligibilityChange` in `EligibilityForm`. I believe if I were to refactor `EligibilityForm` in this manner, I would be breaking its some functionality  in `LandingPage` that was added in PR #10388.